File: mobile/src/main/java/info/papdt/express/helper/ui/fragment/add/StepNoInternetConnection.java
package info.papdt.express.helper.ui.fragment.add;

import android.view.View;

import info.papdt.express.helper.R;
import info.papdt.express.helper.ui.AddActivity;

public class StepNoInternetConnection extends AbsStepFragment {

	@Override
	protected int getLayoutResId() {
		return R.layout.fragment_add_step_no_internet;
	}

	@Override
	protected void doCreateView(View rootView) {
		mButtonBar.setOnLeftButtonClickListener(new View.OnClickListener() {
			@Override
			public void onClick(View view) {
				getAddActivity().onBackPressed();
			}
		});
	}

}

File: mobile/src/main/kotlin/info/papdt/express/helper/ui/fragment/add/StepNoInternetConnection.kt
package info.papdt.express.helper.ui.fragment.add

import android.os.Bundle
import android.view.View

import info.papdt.express.helper.R

class StepNoInternetConnection : AbsStepFragment() {

	override fun getLayoutResId(): Int {
		return R.layout.fragment_add_step_no_internet
	}

	override fun onViewCreated(rootView: View, savedInstanceState: Bundle?) {
		mButtonBar.setOnLeftButtonClickListener(View.OnClickListener { addActivity.onBackPressed() })
	}

}
